src/client/datascience/interactive-window/interactiveWindow.ts

```diff
@@ -510,15 +511,31 @@ export class InteractiveWindow implements IInteractiveWindowLoadable {
             .getCells()
             .find((cell) => getInteractiveCellMetadata(cell)?.id === id);
         if (matchingCell) {
-            this.revealCell(matchingCell, notebookEditor);
+            this.revealCell(matchingCell, notebookEditor, true);
         }
     }
 
-    private revealCell(notebookCell: NotebookCell, notebookEditor: NotebookEditor) {
+    private revealCell(notebookCell: NotebookCell, notebookEditor: NotebookEditor, useDecoration: boolean) {
         const notebookRange = new NotebookRange(notebookCell.index, notebookCell.index + 1);
+        const decorationType = useDecoration
+            ? notebooks.createNotebookEditorDecorationType({
+                  backgroundColor: new ThemeColor('peekViewEditor.background'),
+                  top: {}
+              })
+            : undefined;
         // This will always try to reveal the whole cell--input + output combined
         setTimeout(() => {
             notebookEditor.revealRange(notebookRange, NotebookEditorRevealType.Default);
+
+            // Also add a decoration to make it look highlighted (peek background color)
+            if (decorationType) {
+                notebookEditor.setDecorations(decorationType, notebookRange);
+
+                // Fire another timeout to dispose of the decoration
+                setTimeout(() => {
+                    decorationType.dispose();
+                }, 2000);
```

Comment:
1. Hmm, feels hackis, what if i select another cell or run multiple cells.
Now we'd have multiples with the same style.

2. Why are we disposing this after 2s, why don't we just add this to the list of disposables or another list and clear that (dispose it) when we reveal another cell.

Reply:
That's the behavior from before. Just replicating it. It flashed for 2 seconds.